.circleci/config.yml

```diff
@@ -133,6 +133,88 @@ jobs:
             - store_test_results:
                 path: test/functional/reports
 
+    functional-tests-without-DRM:
+        executor: dashjs-executor
+        steps:
+            - functional_precondition
+            - checkout
+            - dependencies_setup
+            - build_unit_test_steps
+            - functional_test_setup
+            - run:
+                name: Run functional tests (edge / https) VOD (Static MPD)
+                command:
+                    node test/functional/runTests.js --selenium=remote --reporters=junit --app=remote --browsers=edge --protocol=https --streams="VOD (Static MPD)"
+            - run:
+                name: Run functional tests (edge / https) LIVE (Dynamic MPD)
+                when: always # run tests even if some previous tests failed
+                command:
+                    node test/functional/runTests.js --selenium=remote --reporters=junit --app=remote --browsers=edge --protocol=https --streams="LIVE (Dynamic MPD)"
+            - run:
+                name: Run functional tests (edge / https) Live Low Latency
```

Comment:
I assume the different tasks are processed sequentially. Is it possible to run them in parallel? Do we need to split this to separate jobs then?

Reply:
agree, we should use jobs instead of sequential steps! Again, this can be done very elegant using configurable jobs. Jobs ran in a workflow are executed in [parallel](https://circleci.com/blog/decrease-your-build-times-by-running-jobs-in-parallel-with-workflows/)